MAUI/DataGrid/export-to-excel.md

```diff
@@ -0,0 +1,548 @@
+
+
+* CanExportGroups = false;
+
+<img alt="Export DataGrid to Excel format with group summary rows" src="Images\Export-to-Excel\maui-datagrid-datagrid-to-excel.png" width="689"/>
+
+### Export header
+
+By default, the column headers will be exported to Excel sheet. To export the SfDataGrid without column headers, set the `DataGridExcelExportingOption.CanExportHeader` property to `false`.
+
+{% tabs %}
+{% highlight c# %}
+DataGridPdfExportingOption option = new DataGridPdfExportingOption();
+        option.CanExportHeader = false;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format without header cells" src="Images\Export-to-Excel\maui-datagrid-exclude-header.png" width="689"/>
+
+### ExportGroupSummary
+
+By default, the `GroupSummary` rows in the data grid will be exported to Excel. To export the `SfDataGrid` without group summaries, set the `DataGridExcelExportingOption.CanExportGroupSummary` property to `false`.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.CanExportGroupSummary= true;
+option.CanExportGroups = true;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format with group summary rows" src="Images\Export-to-Excel\maui-datagrid-export-group-summary.png" width="689"/>
+
+#### Export table summary
+
+By default, table summaries in the data grid will be exported to Excel. To export the SfDataGrid without table summaries, set the `DataGridExcelExportingOption.CanExportTableSummary` property to `false`.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.CanExportTableSummary= true;
+{% endhighlight %}
+{% endtabs %}
+
+* CanExportTableSummary = true;
+<img alt="Export DataGrid with table summary" src="Images\Export-to-Excel\maui-datagrid-export-table-summary.png" width="689"/>
+
+
+* CanExportTableSummary = false;
+
+<img alt="Export DataGrid without table summary" src="Images\Export-to-Excel\maui-datagrid-exclude-table-summary.png" width="689"/>
+
+
+### Export groups with outlines
+
+To export the data grid with applied grouping, enable the group expand or collapse option in the Excel sheet by setting the `DataGridExcelExportingOption.CanAllowOutlining` to `true`. The default value of this property is false so, you cannot expand or collapse the group in the Excel sheet.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+        option.CanAllowOutlining = true;
+{% endhighlight %}
+{% endtabs %}
+
+### Exclude columns when exporting
+
+By default, all the columns (including hidden columns) in the SfDataGrid will be exported to Excel. To exclude some particular columns when exporting to Excel, add those columns to the `DataGridExcelExportingOption.ExcludeColumns` property in `DataGridExcelExportingOption` list.
```

Comment:
By default, all columns (including hidden columns) in the SfDataGrid are exported to Excel. To exclude specific columns when exporting to Excel, add those columns to the `ExcludeColumns` property in the `DataGridExcelExportingOption` list.

Reply:
updated